apps/modernization-ui/src/apps/patient/add/extended/inputs/race/RaceMultiEntry.spec.tsx

```diff
@@ -46,15 +46,15 @@ describe('RaceMultiEntry', () => {
     });
 
     it('should display proper defaults', async () => {
-        const { getByLabelText, getAllByRole, getAllByText } = render(
+        const { getByLabelText, getAllByRole, getAllByTestId } = render(
             <RaceMultiEntry onChange={onChange} isDirty={isDirty} />
         );
 
         const dateInput = getByLabelText('As of');
         expect(dateInput).toHaveValue(internalizeDate(new Date()));
 
-        const race = getByLabelText('Race');
-        expect(race).toHaveValue('');
+        const race = getAllByTestId('label');
+        expect(race[1]).toHaveTextContent('Race');
```

Comment:
issue:  missed this before, this is just hiding the issue it was warning you about.  There are two elements with the same id in the document.

Reply:
Not clear to me why this change is needed. Was the test failing when using `getByLabelText`?

Reply:
Resolved the merge conflicts i caused and fixed this conflicting id